lib/request-wrapper.ts

```diff
@@ -346,6 +368,40 @@ export class RequestWrapper {
     return this.axiosInstance;
   }
 
+  private static getRetryConfig(
+    axiosInstance: AxiosInstance,
+    retryConfig?: RetryOptions
```

Comment:
suggestion: rename this param to be "retryOptions" to make it clear that it is the set of options that we're exposing to our users

Reply:
Done!